site/docs/components/progress/examples.mdx

```diff
@@ -0,0 +1,38 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  label: Examples
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+
+<LivePreview componentName="component-name" exampleName="ComponentExample" title="Circular progress">
+
+Use linear or circular depending on the context, layout and functionality of an application. Use the same indicator for the same type of process to ensure consistency.
+
+</LivePreview>
+
+<LivePreview componentName="component-name" exampleName="ComponentExample" title="Linear progress">
+
+Use linear or circular depending on the context, layout and functionality of an application. Use the same indicator for the same type of process to ensure consistency.
```

Comment:
does this line need to be repeated?

Reply:
I need to check that with the folks who authored this component (was it you by any chance?). The examples can be viewed individually (this is the default view on the component pages, unless the user flips the "All examples" switch). Since this blurb seemed relevant to both examples, I just copied it for now (I did wonder, whether moving it to the "Usage" tab might be better though), but my intent was to flag this and work with the component authors and content team to decide what to do.

Reply:
okay that makes sense, I wasn't really aware of how the layout would be when writing the word doc.